assets/js/base/components/checkbox-control/style.scss

```
@@ -75,7 +81,8 @@
 	}
 
 	.wc-block-components-checkbox__input[type="checkbox"]:checked,
-	.has-dark-controls .wc-block-components-checkbox__input[type="checkbox"]:checked {
+	.has-dark-controls
+	.wc-block-components-checkbox__input[type="checkbox"]:checked {
```

Was this auto-formatted? I think keeping each selector in one line makes code clearer.
```suggestion
	.has-dark-controls .wc-block-components-checkbox__input[type="checkbox"]:checked {
```

---

Yeah, it looks like my IDE wrapped it due to length.

---

Ok, let's merge it as-is. :+1: